include/spm.h

```diff
@@ -41,6 +48,25 @@ void spm_jump(void);
  */
 void spm_config(void);
 
+/** @brief Set handler which is called on the fault handler on the secure parition.
```

Comment:
What do you think about `which is called by the SPM fault handler` ? I don't we call the SPM `the secure partition`.